cibyl/sources/elasticsearch/api.py

```diff
@@ -129,18 +129,19 @@ def get_builds(self: object, **kwargs: Argument):
                 job.add_build(Build(str(build['_source']['build_id']),
                                     build["_source"]['build_result']))
 
+        if 'last_build' in kwargs:
+            return self.get_last_build(jobs_found)
+
         return jobs_found
 
-    def get_last_build(self: object, **kwargs: Argument):
+    def get_last_build(self: object, builds_jobs: AttributeDictValue):
```

Comment:
You might still need kwargs here, for example if the user passed --last-build --build-status failure